Python/Product/Analysis/LanguageServer/Server.cs

```diff
@@ -81,6 +81,8 @@ private sealed class ReloadModulesQueueItem : IAnalyzable {
         private bool _testEnvironment;
         private ReloadModulesQueueItem _reloadModulesQueueItem;
 
+        private readonly Dictionary<string, ILanguageServerExtension> _extensions;
```

Comment:
Concurrent so we don't have to lock it?

Reply:
Is that easily available in .NET Core without adding more dependencies?

Reply:
We are already using `ConcurrentDictionary` in `Analysis\Interpreter\Ast`